optimum/exporters/openvino/convert.py

```diff
@@ -48,11 +51,7 @@
 )
 
 
-if is_optimum_version(">=", "1.16.99"):
-    from optimum.exporters.onnx.utils import _get_submodels_and_onnx_configs
-
-else:
-    from optimum.exporters.onnx.__main__ import _get_submodels_and_onnx_configs
+init_model_configs()
```

Comment:
shouldn't it be called only once in `model_configs` so that we can overwritte ONNX config in case we want to ?

Reply:
if you can point me place, where it can be called to be triggered once, I can move

Reply:
wouldn't there work https://github.com/eaidova/optimum-intel/blob/ea/ov_export_configs/optimum/exporters/openvino/model_configs.py#L43

Reply:
I'll try to experiment with that

Reply:
reworked